build/commands/lib/config.js

```diff
@@ -171,6 +171,7 @@ const Config = function () {
   this.braveAndroidKeystorePassword = getNPMConfig(['brave_android_keystore_password'])
   this.braveAndroidKeyPassword = getNPMConfig(['brave_android_key_password'])
   this.braveVariationsServerUrl = getNPMConfig(['brave_variations_server_url']) || ''
+  this.braveDebugSymbolsLevel = getNPMConfig(['brave_debug_symbols_level']) || 1
```

Comment:
Possible future value:
0: don't process debug symbols at all (for PRs)
1: the current behaviour 
2: breakpad syms + platform_symbols (pdb, unstripped binaries, etc)

Reply:
I'm not sure this is something that makes sense to configure here. This should not be the source of configuration defaults, it should be the overrides for the defaults set in gn and I think it makes more sense to set `brave_debug_symbols_level = 2` for `is_official_build` and 0 otherwise. We should only override the value if it's set on the command line, but not specify a default value in config.js

Reply:
also this should be command-line, not npm config

Reply:
I guess it could be npm config if we don't want it for PR builds, but should also be a command line option

Reply:
PR's builds are also `is_official_build=true`, therefore we should split them from public builds. That's why `npm config` is used.
As for `a command line option`: ok, I will add it.

Reply:
Added